### PR TITLE
Feature/apmw 7 user data model

### PR DESCRIPTION
**Pull request by bewee**

### CODE REVIEW

**Comment by bewee** [comment]
Reviewing this before merging PRs closing #8  and #9 is probably no fun, since we merged the respective changes into our branch (we will rebase as soon as #8 and #9 are closed).

File: app/models/user.rb [hunk 1]
```diff
@@ -3,30 +3,49 @@ class User < ApplicationRecord
...
       # https://www.rubydoc.info/github/plataformatec/devise/Devise.friendly_token
       user.password = Devise.friendly_token[0, 20]
-      user.username = auth.info.name
-      # `first_name` & `last_name` are also available
-      # user.first_name = auth.info.first_name
-      # user.last_name = auth.info.last_name
-      # If you are using confirmable and the provider(s) you use validate emails,
-      # uncomment the line below to skip the confirmation emails.
+
+      set_auth_data(auth.info.first_name, auth.info.last_name)
```
Comment (joh-dah) [inline app/models/user.rb:30]: Seems like this Method is not yet implemented

File: app/models/user.rb [hunk 2]
```diff
@@ -37,4 +56,17 @@ def self.from_omniauth(auth)
   #     end
   #   end
   # end
+
+  private
+
+  def read_auth_data(auth)
```
Comment (joh-dah) [inline app/models/user.rb:62]: Method not used at the moment

**Review by taminob — APPROVED** [review]
most beautiful code i've ever seen

**Pull request merged**